libbeat/docs/shared-config-ingest.asciidoc

```diff
@@ -34,40 +35,5 @@ output.elasticsearch:
   pipeline: my_pipeline_id
 ------------------------------------------------------------------------------
 
-For example, let's say that you've defined the following pipeline in a file
```

Comment:
I don't agree with removing the example from this topic. We should update it, if necessary, but without the example, I don't think this topic really adds much value for users beyond what they get from reading the config reference.

Reply:
Thanks for the feedback. I verified that the example works and re-added it.